### ISSUE TITLE
Flapjack overwrite

### ISSUE THREAD

**Issue by JMante1** [SynBioDex/Xperimental-Data-Connector-Server#2]
At the moment fj.create is called using confirm=False (it cannot ask for user input on each item if it is going to be used on the server interface). However, this may thus be overwriting all flapjack data. It should be updated so there is an option of overwrite or not as well as confirmation. Then an input for flapjack overwrite can be added to the form too.

**Comment by JMante1** [comment]
@Gonza10V has confirmed that if confirm=False then objects are fully overwritten

**Comment by Gonza10V** [comment]
Yes, sorry for the late response I had a meeting with Tim and he confirmed that, he gave the approval for the refactor of the function according to your use case. I would add the argument overwrite=False or True to use in these cases. The argument confirm is for a more manual use case when you try to create an object to know that it exists and then make the choice of overwrite or get the existing object.
Would the overwrite argument be useful for you?

Gonzalo Vidal
________________________________

**Comment by JMante1** [comment]
Yes. It would be great to have a overwrite=False or True for the case that confirm=False.
